presto-orc/src/main/java/com/facebook/presto/orc/OrcDecompressor.java

```diff
@@ -28,6 +28,12 @@
 
     static Optional<OrcDecompressor> createOrcDecompressor(OrcDataSourceId orcDataSourceId, CompressionKind compression, int bufferSize)
             throws OrcCorruptionException
+    {
+        return createOrcDecompressor(orcDataSourceId, compression, bufferSize, false);
+    }
+
+    static Optional<OrcDecompressor> createOrcDecompressor(OrcDataSourceId orcDataSourceId, CompressionKind compression, int bufferSize, boolean zstdJniDecompressionEnabled)
+            throws OrcCorruptionException
     {
```

Comment:
check state here `zstdJniDecompressionEnabled` is true only if `compression` is `ZSTD`

Reply:
Actually no; no need to check state